### PR TITLE
Allow more recent Eventmachine and Thin gems

### PR DESCRIPTION
**Pull request by sodabrew**
This is intended to unblock https://github.com/sj26/mailcatcher/pull/431/

### CODE REVIEW

**Comment by sj26** [comment]
Have you seen the previous efforts to do this, and tested it thoroughly?

**Comment by sj26** [comment]
i.e. see #4

**Comment by sodabrew** [comment]
I haven't! I'll take a look now.

**Comment by sodabrew** [comment]
... I don't understand what the result of #4 was. Can you summarize what broke and what I should look at?

**Comment by sj26** [comment]
Sorry, I don't remember specifically. But if you run mailcatcher with skinny and new thin/eventmachine then you should be able to receive emails and see the web ui update in real-time via the web socket in all major browsers, and it should work in both foreground and background modes.

**Comment by montdidier** [comment]
Noting that with https://github.com/sj26/skinny/pull/8 and https://github.com/sj26/mailcatcher/pull/443 it seems to work and this is superseded.
